src/dune

```diff
@@ -0,0 +1,48 @@
+(library
+  (name elpi)
+  (public_name elpi)
+  (preprocess (per_module
+    ((pps ppx_deriving.std trace_ppx --trace_ppx-on) elpi_runtime_trace_on)
+    ((pps ppx_deriving.std trace_ppx --trace_ppx-off) elpi_runtime_trace_off)
+    ((action (system "camlp5o -I . -I +camlp5 pa_extend.cmo pa_lexer.cmo %{input-file}")) elpi_parser)
```

Comment:
A bit curious about the necessity of `system` here. You don't seem to be using shell features.

Reply:
I tried with `run`, but it insists in running `camlp5o.opt` that is then unable to load the `cmo`s.